phpunit.xsd

```diff
@@ -118,6 +118,7 @@
         <xs:restriction base="xs:string">
             <xs:enumeration value="default"/>
             <xs:enumeration value="defects"/>
+            <xs:enumeration value="defects,random"/>
```

Comment:
since this order seems to be working as expected (as long as we don't take data-providers into account), it seems there is no need to not call this order as "supported".

fixing the data provider ordering is a separate problem, which also affects `defect` alone, without random ordering afterwards